### PR TITLE
NO-ISSUE: [master] Bump OCP versions: 4.18

### PR DESCRIPTION
**Pull request by danmanor**
OpenShift updates: 4.18.0-rc.2 -> 4.18.0-rc.3, 4.18.0-rc.2-multi -> 4.18.0-rc.3-multi
RHCOS updates: 

/test edge-e2e-metal-assisted-4-18
/cc @rccrdpccl @gamli75

### CODE REVIEW

File: data/default_os_images.json [hunk 1]
```diff
@@ -233,4 +233,4 @@
         "url": "https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/pre-release/4.18.0-rc.2/rhcos-4.18.0-rc.2-s390x-live.s390x.iso",
         "version": "418.94.202411221729-0"
     }
-]
+]
```
Comment (paul-maidment) [inline data/default_os_images.json:236]: Why did this change?

File: data/default_release_images.json [hunk 2]
```diff
@@ -306,4 +306,4 @@
         "support_level": "beta",
         "version": "4.19.0-ec.0-multi"
     }
-]
+]
```
Comment (paul-maidment) [inline data/default_release_images.json:309]: Why did this change?

**Review by gamli75 — APPROVED** [review]

**Comment by danmanor** [comment]
Closing as duplicate of https://github.com/openshift/assisted-service/pull/7138

**Pull request closed without merging**